.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - '0.10'
+before_install:
+  - 'export DISPLAY=:99.0'
+  - 'sh -e /etc/init.d/xvfb start'
+before_script:
+  - 'npm install -qg bower grunt-cli karma-cli'
```

Comment:
you don't need the opening and closing ``'`` - it can just be ``- npm install -qg bower grunt-cli karma-cli``